src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -106,7 +106,7 @@ public class StripePaymentService : IPaymentService
 
             if (taxInfo != null && !string.IsNullOrWhiteSpace(taxInfo.BillingAddressCountry) && !string.IsNullOrWhiteSpace(taxInfo.BillingAddressPostalCode))
             {
-                var taxRateSearch = new TaxRate() 
+                var taxRateSearch = new TaxRate()
```

Comment:
Thanks for cleaning up my bad whitespace. To match code style, please fix more of my mistakes and remove these unnecessary parenthesis.
```suggestion
                var taxRateSearch = new TaxRate
```

Reply:
lol